lib/puppet/provider/iis_application_pool/webadministration.rb

```diff
@@ -59,13 +67,18 @@ def self.prefetch(resources)
   end
 
   def self.instances
-    inst_cmd = ps_script_content('_getapppools', @resource)
+    cmd = []
+    cmd << self.add_powershell2_support if self.ps_major_version == 2
+
```

Comment:
in destroy, the `cmd` building is all together but in instances there's this space...